storage-operator/pkg/controller/volume/volume_controller.go

```diff
@@ -214,25 +214,25 @@ func (r *ReconcileVolume) Reconcile(request reconcile.Request) (reconcile.Result
 	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Commit message:

```diff
- were retaining it);
+ were retaining it).
```

Unless that's intended.

Reply:
Done